src/installer.js

```diff
@@ -119,6 +119,47 @@ module.exports.checkBabel = function checkBabel() {
   this.install(missing);
 };
 
+module.exports.checkESLint = function checkESLint() {
+  try {
+    var options = JSON.parse(
+      fs.readFileSync(
+        path.join(process.cwd(), ".eslintrc"),
```

Comment:
There are more possible formats with extensions for e.g `eslintrc.yml, eslintrc.js` and so forth